Sources/SwiftInspectorCommands/InitializerCommand.swift

```diff
@@ -38,11 +38,11 @@ final class InitializerCommand: ParsableCommand {
   @Option(help: "The name of the type whose initializer information we'll be looking for")
   var name: String
 
-  @Flag(name: .shortAndLong, default: true, inversion: .prefixedEnableDisable, help: typeOnlyHelp)
-  var typeOnly: Bool
+  @Flag(name: .shortAndLong, inversion: .prefixedEnableDisable, help: typeOnlyHelp)
+  var typeOnly: Bool = true
 
   @Option(parsing: .upToNextOption, help: argumentNameHelp)
-  var parameterName: [String]
+  var parameterName: [String] = []
```

Comment:
SwiftArgumentParser before defaulted arrays to an empty array, see https://github.com/apple/swift-argument-parser/blob/main/CHANGELOG.md#changes